File: Tests/260910/task 1/code.cpp
#include <iostream>
#include <string>
#include <conio.h>

int main() 
{
    std::string name;
    unsigned char age;

    std::cout << "Name << ";
    std::cin >> name;

    std::cout << "Age << ";
    std::cin >> age;

    std::cout << "Hello, " << name << "! Your age is " << age << ".\n\n Press any key to exit.\n";

    _getch();

    return 0;
}

File: Tests/260910/task 1/code.cs
namespace TestCS
{
    internal class Program
    {
        static void Main(string[] args)
        {
            Console.Write("Name << ");
            string name = Console.ReadLine();

            Console.Write("Age << ");
            int age = Convert.ToInt32(Console.ReadLine());

            Console.WriteLine($"Hello, {name}! Your age is {age}.\n\nPress ENTER to exit.");
            

            name = Console.ReadLine();
        }
    }
}
